src/client/components/pages/privacy.js

```diff
@@ -20,6 +20,11 @@ const React = require('react');
 
 const PageHeader = require('react-bootstrap').PageHeader;
 
+/**
+* Description of the function PrivacyPage
```

Comment:
"Description of"… This doesn't seem like how other files have been documented. Read some of the other files that already have been documented so that the documentation can at least try to follow the same style and doesn't feel too disjointed.